src/form-builder/CheckboxField.tsx

```diff
@@ -25,7 +25,7 @@ const CheckboxField = (props: CheckboxProps): JSX.Element => {
       checked={field.value}
       onVaChange={(e: CustomEvent) => {
         helpers.setValue((e?.target as HTMLInputElement).checked);
-        if(props.onValueChange) {
+        if (props.onValueChange) {
```

Comment:
This might be something to discuss later or put into our eslint rules if we want this type of formatting to happen by default.